pkgs/development/python-modules/django-lasuite/default.nix

```
@@ -0,0 +1,83 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  django,
+  djangorestframework,
+  joserfc,
+  mozilla-django-oidc,
+  pyjwt,
+  requests,
+  setuptools,
+  wheel,
+  factory-boy,
+  pytest,
+  pytest-django,
+  responses,
+  ruff,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "django-lasuite";
+  version = "0.0.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "suitenumerique";
+    repo = "django-lasuite";
+    tag = "v${version}";
+    hash = "sha256-0UeXMqpk7DPWAGZto7nHhvFK6YsNqywrTFQ/q+7IHDY=";
+  };
+
+  preCheck = ''
+    export PYTHONPATH=tests:$PYTHONPATH
+    export DJANGO_SETTINGS_MODULE=test_project.settings
+  '';
+
+  pythonRelaxDeps = true;
+
+  build-system = [
+    hatchling
+  ];
+
+  dependencies = [
+    django
+    djangorestframework
+    joserfc
+    mozilla-django-oidc
+    pyjwt
+    requests
+  ];
+
+  optional-dependencies = {
+    build = [
+      setuptools
+      wheel
+    ];
+    dev = [
+      factory-boy
+      pytest
+      pytest-django
+      responses
+      ruff
+    ];
+  };
```

```suggestion
  optional-dependencies = {
    dev = [
      factory-boy
      pytest
      pytest-django
      responses
      ruff
    ];
  };
```

- build is probably nothing that any other package would reuse and it is outdated as well
- dev dependencies can just selectively be moved into `nativeCheckInputs` instead
- pytest is redundant with pytestCheckHook
- I doubt that ruff is needed for running the tests